server/mvcc/watchable_store.go

```diff
@@ -329,7 +329,7 @@ func (s *watchableStore) moveVictims() (moved int) {
 //	1. choose a set of watchers from the unsynced watcher group
 //	2. iterate over the set to get the minimum revision and remove compacted watchers
 //	3. use minimum revision to get all key-value pairs and send those events to watchers
-//	4. remove synced watchers in set from unsynced group and move to synced group
+//	4. remove unsynced watchers in set from unsynced group and move to synced group
```

Comment:
I don't think its cleaner. 
When watch becomes "synced" its moved from the unsynced group to synced group.

Reply:
yes！I misunderstood. Thanks.